lib/format/kdbx.js

```diff
@@ -92,6 +92,37 @@ Kdbx.create = function(credentials, name) {
 };
 
 /**
+ * Import database from xml file
+ * If there was an error loading xml file, throws an exception
+ * @param {ArrayBuffer} data - xml file contents
+ * @param {KdbxCredentials} credentials
+ * @param {Function} callback (called with loaded Kdbx)
+ */
+Kdbx.import = function(data, credentials, callback) {
```

Comment:
please, rename to loadXml, like saveXml. import sounds more like merging into an existing db  
also, move this method a bit up, right after load method, before create* section